src/Pretzel.Logic/Configuration.cs

```diff
@@ -80,5 +87,62 @@ public bool TryGetValue(string key, out object value)
         {
             return new Dictionary<string, object>(_config);
         }
+
+        public IDefaultsConfiguration GetDefaults()
+        {
+            return new DefaultsConfiguration(_config);
```

Comment:
Since it is used with the site configuration, is it really necessary to instantiate a new defaults each time?